src/service/poc_lora.proto

```diff
@@ -156,6 +162,12 @@ message lora_verified_witness_report_v1 {
   // the participant to which the reason applies,
   // which rendered the report as invalid
   invalid_participant_side participant_side = 8;
+  /// the transmit gain value of the gateway in dbi x 10
+  /// For example 1 dbi = 10, 15 dbi = 150
+  /// derived from gateway metadata
+  int32 gain = 9;
+  /// The asserted elevation of the gateway, derived from gateway metadata
+  int32 elevation = 10;
```

Comment:
shouldn't this be in the `lora_valid_witness_report`? how does a "verified" witness relate to a "valid" witness?

Reply:
confirmed the valid report for witnesses is deprecated